utils/economic-checks/fuzz/Cargo.lock

```diff
@@ -0,0 +1,3867 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I know the answer, but can't we somehow ignore such huge lock?
Or append it to our main one to reduce diff

Reply:
Including it in our workspace seems ok for me, just exclude it from scripts with --workspace --exclude "crate"